solana/programs/swap-layer/src/state/staged/outbound.rs

```diff
@@ -51,6 +51,9 @@ pub struct StagedOutboundInfo {
     /// The specified token account to refund USDC. This account is required by the Token Router
     /// program in case a fast order is reverted at the Matching Engine after it has been placed.
     pub usdc_refund_token: Pubkey,
+
+    /// Min amount out.
```

Comment:
I think this comment should either be deleted or expanded to explain what it does. Right now it's not adding anything

Reply:
Fair enough, will remove.